src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -70,6 +70,14 @@ internal bool NeedsGeneratedIsByRefLikeAttribute
             }
         }
 
+        internal bool NeedsGeneratedNullableAttribute
```

Comment:
Is this property used?

Reply:
There is one caller, `PEAssemblyBuilder.CreateEmbeddedAttributesIfNeeded`, similar to `NeedsGeneratedIsReadOnlyAttribute` and `NeedsGeneratedIsByRefLikeAttribute`. I've changed the properties to `protected` so it's more obvious where the properties are used.